static/js/dynamic-image-load.js

```diff
@@ -0,0 +1,43 @@
+(function () {
+  var partners = document.querySelectorAll('.js-container');
```

Comment:
`js-container` seems vague. Can you use a more specific class such as `js-partner-logos` or something.